instrumentation/spymemcached-2.12/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spymemcached/SpymemcachedAttributesGetter.java

```diff
@@ -11,37 +11,39 @@
 public class SpymemcachedAttributesGetter implements DbClientAttributesGetter<SpymemcachedRequest> {
 
   @Override
-  public String getSystem(SpymemcachedRequest spymemcachedRequest) {
+  public String getDbSystem(SpymemcachedRequest spymemcachedRequest) {
     return "memcached";
   }
 
+  @Deprecated
   @Override
   @Nullable
   public String getUser(SpymemcachedRequest spymemcachedRequest) {
     return null;
   }
 
-  @Override
   @Nullable
-  public String getName(SpymemcachedRequest spymemcachedRequest) {
+  @Override
+  public String getDbNamespace(SpymemcachedRequest spymemcachedRequest) {
     return null;
   }
 
+  @Deprecated
   @Override
   @Nullable
   public String getConnectionString(SpymemcachedRequest spymemcachedRequest) {
     return null;
   }
 
-  @Override
   @Nullable
-  public String getStatement(SpymemcachedRequest spymemcachedRequest) {
+  @Override
```

Comment:
can you move `@Override` back above `@Nullable` (sorry, just helps a lot to keep diffs down on these big PRs)